### PR TITLE
Fix Mingw string format warning

### PR DESCRIPTION
**Pull request by DanRStevens**
Delegate to recently extracted Format methods from the Log code.

Note: This is based off the Log code updates that extracted the Format methods.

### CODE REVIEW

**Comment by DanRStevens** [comment]
Now that PR #44 has been merged to master, I'm changing the base branch from `updateLoggingCode` to `master`.

**Comment by DanRStevens** [comment]
Merged master to resolve conflicts with recent additions.

File: OPUNetGameSelectWnd.cpp [hunk 1]
```diff
@@ -642,21 +644,23 @@ void OPUNetGameSelectWnd::OnReceive(Packet &packet)
...
+			{
+				// Not quite true, since internal port might be random (no hosting, but possibly still open)
+				//text += " (Direct Host Capable)";
+			}
+			if (bTwoExternal)
+			{
+				text += "\nWarning: Address and Port-Dependent Mapping detected\nYou may have difficulty joining games.";
+			}
+			// Update net info text
+			SendDlgItemMessage(this->hWnd, IDC_NetInfo, WM_SETTEXT, 0, (long)text.c_str());
```
Comment (Brett208) [inline OPUNetGameSelectWnd.cpp:662]: Is there a way to update to a non c-style cast? I don't think static_cast or dynamic_cast would work.

**Review by Brett208 — APPROVED** [review]
Looks good. For maintainability and readability, we need to eventually break `OPUNetGameSelectWnd::OnReceive` into smaller chunks.

File: OPUNetGameSelectWnd.cpp (hunk 1 above)
Comment (DanRStevens) [reply OPUNetGameSelectWnd.cpp:662]: For sure we can use C++ style casts. I'll investigate in a future branch. There are a lot of similar C-style casts. I figure I should try to handle them all at once.

**Comment by DanRStevens** [comment]
Agreed on eventually breaking that method into smaller chunks. That scope block is a huge indication of that.

----

Tagging Issue #17. This removes the use of `strncat`, which could not be removed in PR #47.

**Pull request merged**